ios/RCTContacts/RCTContacts.m

```diff
@@ -468,6 +470,16 @@ -(void) updateRecord:(CNMutableContact *)contact withData:(NSDictionary *)contac
         else if ([label isEqual: @"iPhone"]){
             phone = [[CNLabeledValue alloc] initWithLabel:CNLabelPhoneNumberiPhone value:[[CNPhoneNumber alloc] initWithStringValue:number]];
         }
+        // Adding the following label types because the a field's baseField doesn't match the defined types. This will safeguard us from exchange containers
+        else if ([label isEqual: @"cellPhone"]){
+            phone = [[CNLabeledValue alloc] initWithLabel:CNLabelPhoneNumberMobile value:[[CNPhoneNumber alloc] initWithStringValue:number]];
+        }
+        else if ([label isEqual: @"fax"]){
+            phone = [[CNLabeledValue alloc] initWithLabel:CNLabelPhoneNumberOtherFax value:[[CNPhoneNumber alloc] initWithStringValue:number]];
+        }
+        else if ([label isEqual: @"phone"]){
```

Comment:
@JulesPatry office phones base field is 'phone'. I could switch to using the display type on the field, but we are going to me limited. There is not "ext" label because adding a extension is not common, so they are hitting a default option which is other.